bayesian_testing/experiments/delta_normal.py

```diff
@@ -0,0 +1,298 @@
+from numbers import Number
+from typing import List, Tuple
+import numpy as np
+from bayesian_testing.experiments.base import BaseDataTest
+from bayesian_testing.metrics import eval_delta_normal_agg
+from bayesian_testing.utilities import get_logger
+
+logger = get_logger("bayesian_testing")
+
+
+class DeltaNormalDataTest(BaseDataTest):
+    """
+    Class for Bayesian A/B test for Delta-Normal data (Normal with non-positive values).
```

Comment:
IIUC Delta Normal Test will be for cases where we want to handle couple of 0 values with Normally distributed data. To do so, we handle zero vs non-zero with Beta distribution and non-zero values with Normal. So it is similar to Lognormal but I think the wording should be more in a direction: "Normally distributed conversions". Because Normal with non-positive sounds tricky.